subprojects/common/statsd/src/test/kotlin/com/avito/android/stats/SeriesNameTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.avito.android.stats
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.Test
+
+internal class SeriesNameTest {
+
+    @Test
+    fun `create series name - success - multiple parts`() {
+        val result = SeriesName.create("one", "two", "three")
+
+        assertThat(result.toString()).isEqualTo("one.two.three")
+    }
+
+    @Test
+    fun `create series name - undesired behavior - multiple parts`() {
+        val result = SeriesName.create("one", "two.three")
+
+        assertThat(result.toString()).isEqualTo("one.two_three")
+    }
+
+    @Test
+    fun `create series name - success - single part`() {
+        val result = SeriesName.create("one")
+
+        assertThat(result.toString()).isEqualTo("one")
+    }
+
+    @Test
+    fun `create series name - keeps underscore`() {
+        val result = SeriesName.create("one_a")
+
+        assertThat(result.toString()).isEqualTo("one_a")
+    }
+
+    @Test
+    fun `create series name - keeps minus`() {
+        val result = SeriesName.create("one-a")
+
+        assertThat(result.toString()).isEqualTo("one-a")
+    }
+
+    @Test
+    fun `create series name - replaces unsupported symbol (percent) with underscore`() {
```

Comment:
Does regex replaces all invalid consecutive symbols by one `_`?
If so, we can check it also in tests

Reply:
thanks, new testcase added

Reply:
I think we can merge these cases into one:
`create series name - replaces unsupported symbols with an underscore`
It includes implicitly `replaces unsupported symbol with underscore`